client/src/AddVideo.js

```diff
@@ -0,0 +1,97 @@
+import React, { useEffect, useState } from 'react';
+import './AddVideo.css';
+
+function AddVideo({ videos, setVideos }) {
+    const [clickAdd, setClickAdd] = useState(false);
+    const[titleInput, setTitleInput] = useState("");
+    const[urlInput, setUrlInput] = useState("");
+    
+    const handleAddVideoButton = () => {
+        //clickAdd === false ? setClickAdd(true) : setClickAdd(false);
+        setClickAdd(!clickAdd);
+    };
+
+    const handleTitleInputChange = (e) => {
+       // e.preventDefault();
+        setTitleInput(e.target.value);
+    };
+
+    const handleUrlInputChange = (e) => {
+        //e.preventDefault();
+        let url = e.target.value;
+        if(url.includes('https://www.youtube.com/')){
+          url = url.replace('watch?v=', 'embed/');
+            return setUrlInput(url);
+        }else{
+            alert('Invalid URL format. Please provide a YouTube URL!');
```

Comment:
Ideally  the error message will be in the page as a HTML element, better for usability and accessibility